integration/suite.rb

```diff
@@ -72,11 +72,7 @@ def pink
       puts "#{test_name} #{'SUCCEEDED'.green}"
     ensure
       if options[:vm_maintenance] && succeeded
-        puts "Halting VM".yellow
-        unless system("vagrant halt")
-          puts "Tried to halt #{test_name} but it failed".red
-        end
-        puts "Halted VM"
+        
```

Comment:
Halting the VM doesn't clean up at all (you need to destroy it to actually clean it up). My initial goal with halting the VM after the test was to save memory as you moved onto the next test.

I'm happy to revert this change, it was convenient for testing the balancer locally to have a VM with a functioning preparer on it. It turns out that halting the VM with a standalone Consul instance causes Consul to lose all of the registered services in the catalog (this does not happen with Consul instances in a quorum) which makes the balancer fail tremendously.